tests/generation/test_utils.py

```diff
@@ -2439,30 +2441,6 @@ def test_contrastive_search_batched(self):
         max_score_diff = (output_sequences_batched.scores[0][1] - output_sequences.scores[0][0]).abs().max()
         self.assertTrue(max_score_diff < 1e-5)
 
-    def test_generate_from_input_embeds_decoder_only(self):
```

Comment:
(this was a bad merge with main in a previous PR -- it is an old version of `test_generate_from_inputs_embeds_decoder_only` on L2492)